src/components/Footer/Footer.js

```diff
@@ -1,48 +1,42 @@
 import React from 'react';
-import { Image } from 'rebass';
-import mapicon from './map-pin.png';
+import { Link } from 'rebass';
 import styled from 'styled-components';
 
-const Copyright = styled.p`
+const FooterText = styled.p`
   color: white;
   font-size: 1.3rem;
   text-align: center;
   margin-bottom: 2vh;
   margin-top: 0;
 `;
 
-const FooterWrapper = styled.div`
-  margin-top: 1.5rem;
-  display: flex;
-  justify-content: flex-end;
-  align-items: center;
-  flex-flow: column nowrap;
-`;
-
-const MapIcon = styled(Image)`margin-bottom: 0.5rem;`;
-
-const LocationText = styled.p`
+const ProjectLink = styled(Link)`
+  text-decoration: none;
   color: white;
-  font-size: 1.6rem;
+  font-size: 1.5rem;
   text-align: center;
-  padding-left: 0.2rem;
-  margin: 0 0 0.5rem 0;
+  margin-bottom: 2vh;
+  margin-top: 0;
+  &:hover {
+    text-decoration: underline;
+  }
 `;
 
-const LocationWrapper = styled.div`
+const FooterWrapper = styled.div`
+  margin-top: 1.5rem;
   display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
+  justify-content: flex-end;
   align-items: center;
+  flex-flow: column nowrap;
 `;
 
 const Footer = () => (
   <FooterWrapper>
-    <LocationWrapper>
-      <MapIcon height="1.6rem" width="1.6rem" src={mapicon} />
-      <LocationText children="Berlin" />
-    </LocationWrapper>
-    <Copyright children="Copyright © 2017" />
+    <FooterText>
+      <ProjectLink href="https://github.com/gymnasion/lyre" children="Github" />{' '}
+      | <ProjectLink href="http://gymnasion.io/" children="Gymnasion" />
```

Comment:
Why you need this?

Reply:
Because I need two separate links that are above copyright line.